VERSION

```diff
@@ -10,9 +10,6 @@
         "v1.9.7"
     ],
     "flannel-cni": [
-        "v0.0.1"
-    ],
-    "nginx-proxy": [
-        "v0.0.1"
+        "v0.0.2"
     ]
```

Comment:
Do we still need to remove this if we don't remove nginx proxy in old version?

Reply:
I am going to use host running nginx instead of container running to avoid https://github.com/rancher/rancher/issues/16074